samples/snippets/csharp/concepts/methods/async1.cs

```diff
@@ -1,19 +1,12 @@
 ﻿// <Snippet102>
-using System;
-using System.Diagnostics;
-using System.Threading.Tasks;
-
-public class Example
+public static class Example
 {
     // This Click event is marked with the async modifier.
-    public static void Main()
-    {
-       DoSomethingAsync().Wait();
-    }
+    public static void Main() => DoSomethingAsync().Wait();
 
     private static async Task DoSomethingAsync()
     {
-        int result = await DelayAsync();
+        var result = await DelayAsync();
```

Comment:
@BillWagner do we want to use `var` here or the concrete types?

E.g. in dotnet/runtime it must be the concrete type, except it's clearly visible what type it is.

Reply:
Yes, it should be the concrete type. I missed that in my review.